docs-v2/integrations/all/servicenow.mdx

```diff
@@ -14,32 +15,117 @@ import PreBuiltUseCases from "/snippets/generated/servicenow/PreBuiltUseCases.md
 
-<Note>Contribute improvements to the setup guide by [editing this page](https://github.com/nangohq/nango/tree/master/docs-v2/integrations/all/servicenow.mdx)</Note>
-
 
 ## Useful links
 
--   [How to register an Application](https://docs.servicenow.com/bundle/vancouver-application-development/page/build/pipelines-and-deployments/task/create-oauth-api-endpoints-for-external-clients.html)
--   [OAuth-related docs](https://docs.servicenow.com/bundle/vancouver-platform-security/page/administer/security/concept/c_OAuthApplications.html)
--   [List of OAuth scopes](https://docs.servicenow.com/bundle/tokyo-application-development/page/administer/security/task/t_SpecifyAnOAuthScope.html)
+| Topic | Links | 
+| - | - | 
+| General | [ServiceNow Developer Portal](https://developer.servicenow.com/dev.do) |
+| | [ServiceNow Documentation](https://docs.servicenow.com/) |
+| | [ServiceNow Community](https://community.servicenow.com/) |
+| OAuth | [OAuth Applications](https://docs.servicenow.com/bundle/vancouver-platform-security/page/administer/security/concept/c_OAuthApplications.html) |
+| | [Create OAuth API Endpoints](https://docs.servicenow.com/bundle/vancouver-application-development/page/build/pipelines-and-deployments/task/create-oauth-api-endpoints-for-external-clients.html) |
+| | [OAuth Scopes](https://docs.servicenow.com/bundle/tokyo-application-development/page/administer/security/task/t_SpecifyAnOAuthScope.html) |
+| API | [REST API Documentation](https://developer.servicenow.com/dev.do#!/reference/api/vancouver/rest/c_TableAPI) |
+| | [API Explorer](https://developer.servicenow.com/dev.do#!/reference/api-explorer) |
+| | [API Best Practices](https://developer.servicenow.com/dev.do#!/guides/vancouver/now-platform/api-best-practices/api-best-practices-overview) |
+
+## Common Scopes
 
-<Note>Contribute useful links by [editing this page](https://github.com/nangohq/nango/tree/master/docs-v2/integrations/all/servicenow.mdx)</Note>
+| Scope | Description |
+| ----- | ----------- |
+| admin | Full administrative access |
+| user_admin | User administration capabilities |
+| user | Access to user data |
+| web_service | Access to web services |
+| personalize | Personalization capabilities |
+| useraccount | Access to user account data |
 
 ## API gotchas
 
--   For security purposes, access tokens will expire after 30 minutes by default, when this happens you can use `nango.getConnection()` to generate a new set of tokens.
+- ServiceNow instances have unique URLs in the format `https://[your-instance-name].service-now.com/`. You must use your specific instance URL when configuring OAuth endpoints.
+
+- ServiceNow's REST API is table-based. Most API calls follow the pattern: `https://[your-instance-name].service-now.com/api/now/table/[table_name]`.
+
+- When making API calls, you must include the `Accept` header with the value `application/json` to receive JSON responses.
```

Comment:
These aren't useful information for the user since the calls are already abstracted by Nango